R/derive_vars_query.R

```diff
@@ -193,7 +196,86 @@ derive_vars_query <- function(dataset, dataset_queries) { # nolint: cyclocomp_li
     group_by_at(static_cols) %>%
     summarise_all(~ first(na.omit(.))) %>%
     ungroup()
```

Comment:
The results should not be assigned to a variable, i.e., `joined <- ` should be removed. Otherwise the output dataset is not printed.